packages/microservices/test/client/client-kafka.spec.ts

```diff
@@ -218,6 +218,20 @@ describe('ClientKafka', () => {
 
       expect(logCreatorSpy.called).to.be.true;
     });
+
+    it('should allow an empty postfixId', async () => {
+      const client = new ClientKafka({
+        postfixId: ''
+      });
+      expect(client.getClientId()).to.eq('nestjs-consumer');
+      expect(client.getGroupId()).to.eq('nestjs-group');
+    });
+
+    it('should postfix clientId and groupId with postfixId', async () => {
```

Comment:
Perhaps this should say

```suggestion
    it('should postfix clientId and groupId with default postfixId', async () => {
```

instead?